.surge.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+set -x
+
+if [ "$TRAVIS_EVENT_TYPE" = "push" -a "$TRAVIS_BRANCH" = "master" ]; then
```

Comment:
This feels wrong. Why would I check Travis variables in a Surge script? The Travis checks should be done in the Travis script.